percona-server-8.0/Dockerfile

```diff
@@ -108,6 +108,17 @@ RUN set -eux; \
 ENV CALL_HOME_OPTIONAL_PARAMS=" -s ${OS_VER}"
 
 COPY ps-entry.sh /docker-entrypoint.sh
+COPY telemetry-agent-supervisor.sh /usr/bin/
```

Comment:
do we need both phase 0 and phase 1?

Reply:
Yes, according to our earlier discussions, we need both phase-0 and phase-1, both have been included in the telemetry-agent-supervisor script.  @maxkondr can provide more information on this.